scripts/build-minified.js

```diff
@@ -1,14 +1,32 @@
 import esbuild from "esbuild"
 import fs from "fs"
 import zlib from "zlib"
+import { globby } from "globby"
 
 
 const globalName = "webnative"
-const outfile = "dist/index.umd.min.js"
-const outfileGz = `${outfile}.gz`
 
-// From https://github.com/umdjs/umd/blob/36fd1135ba44e758c7371e7af72295acdebce010/templates/returnExports.js
-const umd = {
+
+console.log("📦 Bundling & minifying...")
+
+await esbuild.build({
+    entryPoints: ["src/index.ts"],
+    outdir: "dist",
+    bundle: true,
+    splitting: true,
+    minify: true,
+    sourcemap: true,
+    platform: "browser",
```

Comment:
will / should this work in a web worker?

i tried
```js
const webnativeCDNURL = 'https://unpkg.com/webnative@0.34.0-alpha1//dist/index.umd.min.js'
const importRes = self.importScripts(webnativeCDNURL)
```

but i get an `ReferenceError: document is not defined`

Reply:
Ah good point, we totally forgot about that. Will fix 👍  Thanks!